tictactoe.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
Any particular reason for doing this all in one `html` file and not separating concerns by linking to `js` and `css` files